Build prplMesh in QEMU with Yocto Poky 3.1

MuteSpirit opened the pull request:
Add build prplmesh for QEMU in Yocto Poky 3.1
    That's an intermediate step between
    RDKB (OE 2.2) for RPi and RDKB (OE 3.1) for Turris Omnia

    conf/layer.conf: mark support Morty and Dunfell
    conf/poky-dunfell: template build conf files
    prplmesh.bb: move "readline" from RDEPENDS into DEPENDS
    kernel bbappend: rename to be applied for yocto 5.x too

    README.md: Add separate section for Yocto Poky 3.1
    In comparison to Yocto 2.2:
    * fetching "meta-python2"
    * use "dunfell" branch
    * use "conf/poky-dunfell" template conf folder

https://jira.prplfoundation.org/browse/PPM-240

Commit:
prplmesh: build in Yocto Poky 3.1

Add build prplmesh for QEMU in Yocto Poky 3.1
That's an intermediate step between
RDKB (OE 2.2) for RPi and RDKB (OE 3.1) for Turris Omnia

conf/layer.conf: mark support Morty and Dunfell
conf/poky-dunfell: template build conf files
prplmesh.bb: move "readline" from RDEPENDS into DEPENDS
kernel bbappend: rename to be applied for yocto 5.x too

Signed-off-by: Ivan Efimov <i.efimov@inango-systems.com>

Commit:
README.md: add howto Yocto Poky 3.1 build manual

Add separate section for Yocto Poky 3.1
In comparison to Yocto 2.2:
* fetching "meta-python2"
* use "dunfell" branch
* use "conf/poky-dunfell" template conf folder

Signed-off-by: Ivan Efimov <i.efimov@inango-systems.com>

VladyslavTupikin commented:
@MuteSpirit  You can link the Jira issue with Github PR by adding the Jira link in the PR description.

Commit:
prplmesh.bb: shift to prplMesh commit where patches are already upstreamed

Switch to prplMesh commit available on 16 Jul 2020

	commit d55b727b52158d46b3a0263734445ac29116220f
	Author: Moran Shoeg <moranx.shoeg@intel.com>
	Date:   Thu Jul 16 09:05:07 2020 +0000

	    agent: Add/Fix prints on monitor thread

	    Signed-off-by: Moran Shoeg <moranx.shoeg@intel.com>

Remove now unneeded "recipes-connectivity/prplmesh/prplmesh/*.patch" files

Signed-off-by: Ivan Efimov <i.efimov@inango-systems.com>

Commit:
Merge pull request #23 from prplfoundation/feature/PPM-281-patches-are-not-needed-more

prplmesh.bb: shift to prplMesh commit where patches are already upstreamed

MuteSpirit commented:
I've rebased on `morty` branch
